<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_com</class>
 <widget class="QMainWindow" name="user_com">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户评价</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	height:24px;
	width:24px;
	border-image: url(:/off.jpg);
}
QPushButton::checked
{
	border-image: url(:/on.jpg);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	height:24px;
	width:24px;
	border-image: url(:/off.jpg);
}
QPushButton::checked
{
	border-image: url(:/on.jpg);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	height:24px;
	width:24px;
	border-image: url(:/off.jpg);
}
QPushButton::checked
{
	border-image: url(:/on.jpg);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	height:24px;
	width:24px;
	border-image: url(:/off.jpg);
}
QPushButton::checked
{
	border-image: url(:/on.jpg);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>非常好</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	height:24px;
	width:24px;
	border-image: url(:/off.jpg);
}
QPushButton::checked
{
	border-image: url(:/on.jpg);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>请对我们的服务进行评价</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_content">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}</string>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-10</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo_trans.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-60</y>
      <width>541</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/user_com.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>pushButton1</zorder>
   <zorder>pushButton2</zorder>
   <zorder>pushButton5</zorder>
   <zorder>pushButton4</zorder>
   <zorder>labelTitle</zorder>
   <zorder>pushButton3</zorder>
   <zorder>label</zorder>
   <zorder>textEdit_content</zorder>
   <zorder>pushButton_submit</zorder>
   <zorder>pushButton_submit_2</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
